app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
I meant ... the data to speaker-list itself would be passed in from `app/templates/public/index.hbs`